### ISSUE TITLE
INPC003 should not warn for instance property when in static context

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/PropertyChangedAnalyzers#176]
```cs
        public static Settings FromResource
        {
            get
            {
#pragma warning disable INPC003 // Notify when property changes.
                InnerInstance = JsonConvert.DeserializeObject<Settings>(Properties.Resources.Units, Persister.SerializerSettings);
#pragma warning restore INPC003 // Notify when property changes.
                return InnerInstance;
            }
        }
```
